FlowCrypt/Controllers/Setup/SetupViewController.swift

```diff
@@ -353,6 +358,30 @@ extension SetupViewController {
     }
 }
 
+
+// MARK: - Alerts
+extension SetupViewController {
+    private func awaitUserPassPhraseEntry(title: String) -> Promise<String?> {
```

Comment:
This is supposed to be a general method, not just for setup. One day we will have an option to not remember pass phrase by the device, or only remember it temporarily. Then we will need this method to enter pass phrase during message decryption. This is why it was in a general class, I would not put it there otherwise.